homeassistant/components/honeygain/translations/en.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
This file can be removed as it will be generated by CI

Reply:
Ahh Okay awesome, I didn't realise it was generated